### DISCUSSION TITLE
Can't authenticate suddenly

### DISCUSSION THREAD

**Discussion by jlin816** [community/community#8038] [Copilot Conversations]
I've had access to Copilot for a month now, but when I opened VSCode today it errored on authentication: `[ERROR] [default] [2021-11-23T17:05:09.539Z] GitHub Copilot could not connect to server. Extension activation failed: "Forbidden"`. I tried uninstalling and re-installing, re-authenticating Github in VSCode, but it still shows the same message. Any idea how I might fix this?

**Comment by VanBuren77** [comment]
I am also having this issue

**Reply by davecheney** [reply]
Please try visiting https://github.com/settings/copilot and completing the required setup steps.
